tests/unit/scripts/test_lint.py

```diff
@@ -6,7 +6,8 @@
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-#
+#PID: 12632 Threads: 1 RSS: 1044
```

Comment:
?
```suggestion
```

Reply:
I suspect that I may need to fiddle with my mouse-wheel-button settings... I think I'm pasting whilst scrolling...